Embedded_Weaviate.ipynb

```diff
@@ -0,0 +1,221 @@
+{
```

Comment:
Please see the comments on the Docker based file as I think they apply here also.